### PR TITLE
Update config and docs for MP v0.3.1

### PR DESCRIPTION
**Pull request by dennyverbeeck**
Update configuration and documentation for the upcoming MP v0.3.1 release.

### CODE REVIEW

File: dcompose-stack/radar-cp-hadoop-stack/etc/env.template [hunk 1]
```diff
@@ -1,6 +1,8 @@
 SERVER_NAME=localhost
 MANAGEMENTPORTAL_KEY_DNAME=CN=localhost,OU=MyName,O=MyOrg,L=MyCity,S=MyState,C=MyCountryCode
 MANAGEMENTPORTAL_FRONTEND_CLIENT_SECRET=
+MANAGEMENTPORTAL_CATALOGUE_SERVER_ENABLE_AUTO_IMPORT=
+MANAGEMENTPORTAL_CATALOGUE_SERVER_SERVER_URL=
```
Comment (yatharthranjan) [inline dcompose-stack/radar-cp-hadoop-stack/etc/env.template]: Is the `SERVER` word twice intentional here?

Comment (nivemaham) [reply dcompose-stack/radar-cp-hadoop-stack/etc/env.template]: yes

Comment (dennyverbeeck) [reply dcompose-stack/radar-cp-hadoop-stack/etc/env.template]: Yes it is, it's a result of the property hierarchy (`managementportal.catalogeServer.serverUrl`) and Spring's rules for relaxed binding.

Comment (yatharthranjan) [reply dcompose-stack/radar-cp-hadoop-stack/etc/env.template]: Ah. Okay nice

**Pull request merged**